README.md

```diff
@@ -8,30 +8,26 @@ as possible of [HTTPie's](https://httpie.io/) excellent design.
 
 ## Installation
 
-### On macOS and Linux via Homebrew
-```
-brew install xh
-```
+### via curl (Linux/macOS)
 
-### On windows via Scoop
 ```
-scoop install xh
+curl -sfL https://git.io/xh/install.sh | sh

```

Comment:
Not sure why this doesn't work even when I added `install.sh` to master. The link was created using https://github.com/krrishd/gitio
```
$ gitio 'https://raw.githubusercontent.com/ducaale/xh/master/install.sh' 'xh/install.sh'

Congratulations! You can access your URL at https://git.io/xh/install.sh

```

Reply:
Reverted to using `https://raw.githubusercontent.com/ducaale/xh/master/install.sh`